eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.test.js

```diff
@@ -126,131 +134,173 @@ function defaultSetup() {
 }
 
 describe("Destination Picker Menu", () => {
-  it("should default to current section tab", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText("Question one")).toBeVisible();
-  });
+  describe("Hub is disabled", () => {
+    beforeEach(() => {
+      questionnaire = { hub: false };
+      useQuestionnaire.mockImplementation(() => ({ questionnaire }));
+    });
 
-  it("should show active styling for current tab", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-  });
+    it("should default to current section tab", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText("Question one")).toBeVisible();
+    });
 
-  it("should display question pages in 'Current section'", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-    expect(getByText("Question one")).toBeVisible();
-    expect(getByText("Question two")).toBeVisible();
-    expect(getByText("Question three")).toBeVisible();
-    expect(getByText("Question four")).toBeVisible();
-  });
+    it("should show active styling for current tab", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText(tabTitles.current)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
+    });
```

Comment:
It would be better if you asserted that the current tab is active, for instance if it has a certain prop or class to indicate that it's active, rather than asserting the actual styles themselves. Or, testing the indicator as well as the styles.

The styles aren't integral to the functionality and may change throughout the lifespan of the application, so these tests would break whenever there's a redesign. Where, the thing indicating that the tab is active will likely stay the same so we can have confidence that the test won't need to be revised at a later date.